packages/combobox/docs/overview.mdx

```diff
@@ -97,10 +97,12 @@ Use the `loadOptions` prop when you need to populate options from a network requ
 };
 ```
 
-### Required
+## Required
 
 The `Combobox` component will always append `(optional)` to the label based on the `required` prop.
 
+Use the `hideOptionalLabel` prop in situations where you want to indiciate to screen reader users that a field is optional, but don't want to show the "optional" label. When this prop is `true`, "optional" will never be appended to the label, even when `required` is `false`.
```

Comment:
This should only be used for inputs used to filter information from a search. This should never be used for forms that collect user information

Reply:
Good call, updated 👍